bytelatent/model/local_models.py

```diff
@@ -16,29 +17,69 @@
     RotaryEmbedding,
     TransformerBlock,
 )
-from bytelatent.model.transformer import CrossAttention
+from bytelatent.model.global_transformer import CrossAttention
 from bytelatent.model.utils import create_causal_mask, downsample
 from bytelatent.tokenizers.blt_tokenizer import BOE_ID
 
 logger = logging.getLogger()
 
 
+class LocalModelArgs(BaseModel):
+    model_config = ConfigDict(extra="forbid")
```

Comment:
It seems like this could be simplified by inheriting from the BaseTransformerArgs. There should be very few additional things that the local models need to know about.

Reply:
Agreed, assuming that its possible to override the defualt args in `BaseTransformerArgs`